server/impl/src/main/java/com/walmartlabs/concord/server/security/ldap/Utils.java

```diff
@@ -0,0 +1,47 @@
+package com.walmartlabs.concord.server.security.ldap;
+
+/*-
+ * *****
+ * Concord
+ * -----
+ * Copyright (C) 2017 - 2021 Walmart Inc.
+ * -----
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =====
+ */
+
+public class Utils {
+    public static final int maxLevel = 20;
+
+    public static void recursionLimiter() {
+        if (maxLevel == 0)
```

Comment:
I think this condition is never true?

Reply:
yes. just added logic if we don't want recursion at all then setting `maxLevel = 0` will work. not necessary?